crates/wasmparser/src/operators_validator.rs

```diff
@@ -1654,8 +1681,6 @@ impl OperatorValidator {
             | Operator::I32x4Neg
             | Operator::I64x2Abs
             | Operator::I64x2Neg
-            | Operator::I32x4TruncSatF32x4S
```

Comment:
existing: do there need to be check for non-deterministic?

Reply:
AFAIK the deterministic feature isn't heavily used, so yeah I think it's fine to move these to requiring determinism.